.github/workflows/on-push-master.yml

```diff
@@ -0,0 +1,56 @@
+name: Run coveralls and publish to gh-pages
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout sources
+      uses: actions/checkout@v2
+
+    - name: Use Node.js 14.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14.x
+
+    - name: Cache Node.js modules 💾
+      uses: actions/cache@v2
+      with:
+        path: ~/.npm
+        key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.OS }}-node-
+          ${{ runner.OS }}-
+
+    - name: Install dependencies ⏬
+      run: npm install
+
+    - name: Cleanup and prepare pages
+      run: |
+        rm -rf master
+        mkdir -p master/docs
+        mkdir -p master/docs-w-ext
+        cp -r examples master/
+        cp -r resources master/
+        cp -r src master/
+        npm run generate:docs:master
```

Comment:
These need `jsduck`, unfortunately. I'll comment in package.json below.

Reply:
Done in 5e6e76c1